source/policies.ttl

```diff
@@ -32,3 +32,13 @@ Det är du som användare som ansvarar för att ta reda på om materialet är up
 """@sv ;
     :seeAlso <https://www.kb.se/besok-och-anvand/kopiera-och-fotografera/upphovsratt/> ;
     :publisher <https://libris.kb.se/20dgd3zl56pmrzk#it> .
+
+</policy/mtm/encryption-of-course-literature> a :AccessPolicy ;
+skos:prefLabel "Filer kan krypteras vid nedladdning."@sv, "Files may be encrypted during download."@en ;
```

Comment:
Jag har tänkt (för) mycket på detta och förslår att vi ändrar den engelska översättningen till "Files may be encrypted upon download." :)